.ci/check-commit.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+set -e
+
+SHELL_FOLDER=$(
+    cd $(dirname $0)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5569463)